docs/installation.rst

```diff
@@ -1,52 +1,45 @@
-===============================
-Getting Started Guide for Linux
-===============================
+.. meta::
+   :description: A high-performance HIP library for tensor primitives
+   :keywords: hipTensor, ROCm, library, API, tool, installation
 
-------------
-Introduction
-------------
+.. _installation:
 
-This document contains instructions for installing, using, and contributing to hipTensor.
-The quickest way to install is to build from source. The document also contains an API Reference Guide, Programmer's Guide, and Contributor's Guides.
-
-Documentation Roadmap
-^^^^^^^^^^^^^^^^^^^^^
-The following is a list of hipTensor documents in the suggested reading order:
+===============================
+Installation
+===============================
 
- - Getting Started Guide (this document): Describes how to install and configure the hipTensor library; designed to get users up and running quickly with the library.
- - API Reference Guide : Provides detailed information about hipTensor functions, data types and other programming constructs.
- - Programmer's Guide: Describes the code organization, Design implementation detail and those that should be considered for new development and Testing & Benchmarking detail.
- - Contributor's Guide : Describes coding guidelines for contributors.
+This document provides instructions for installing and configuring hipTensor library on Linux.
+The quickest way to install is to build from source.
 
 -------------
 Prerequisites
 -------------
 
 -  A ROCm enabled platform, more information `here <https://rocm.github.io/>`_.
```

Comment:
This link does not work. Which is the correct link for this? https://github.com/ROCm doesn't make sense either.